Source/WebCore/dom/Node.cpp

```diff
@@ -1788,6 +1788,27 @@ void Node::setTextContent(String&& text)
     ASSERT_NOT_REACHED();
 }
 
+std::optional<std::variant<String, RefPtr<TrustedScript>>> Node::textContentForBinding()
+{
+    return textContent();
+}
+
+ExceptionOr<void> Node::setTextContentForBinding(std::optional<std::variant<String, RefPtr<TrustedScript>>> value)
```

Comment:
This is not custom binding code. We just have a version defined for JS API.

Reply:
Is the forBinding make still correct for the method?

Reply:
Yes.